src/ubuntu/24.10/helix/Dockerfile

```diff
@@ -1,32 +1,31 @@
 FROM ubuntu.azurecr.io/ubuntu:oracular AS venv
 ENV DEBIAN_FRONTEND=noninteractive
 
-RUN apt-get update && \
-    apt-get install -y \
+RUN apt-get update \
+    && apt-get install -y \
         cargo \
         pkg-config \
         libffi-dev \
         coreutils \
         python3-dev \
         python3-pip \
         python3-venv \
-        libssl-dev \
-    && rm -rf /var/lib/apt/lists/*
```

Comment:
This cleanup should be added back.

Reply:
It has no benefit. It is in a stage that doesn't contribute to the end product.

Reply:
It is beneficial IMO for two reasons.  First, the cleanup is a Dockerfile best practice.  Folks copy these patterns all over so following the best practices everywhere is beneficial.  Second, from a perf perspective you are still writing the cache in the layer.  In my experience deleting the cache is faster that writing it in the layer.  Admittingly this is trivial.